### ISSUE TITLE
Work with only a few samples

### ISSUE THREAD

**Issue by guillaumejaume** [maragraziani/interpretAI_DigiPath#5]
If as we discussed in the morning you directly download a few samples, I assume that the batch generator is not needed anymore

Also, more of a question in:
```
vgg_histo_model.compile(loss=tf.keras.losses.binary_crossentropy, optimizer=tf.keras.optimizers.SGD(lr=1e-4), metrics=['accuracy'])
```
do you actually need to define a loss function, an optimizer etc. when you only want to do inference? I would expect that specifying these parameters is not needed (but I'm not a Keras user... so idk)

**Comment by maragraziani** [comment]
Yup, fixed. Yes indeed, that needed some cleaning
